### PR TITLE
Update Citadel CLI

### PR DESCRIPTION
**Pull request by pwltr**
Some small updates and fixes

### CODE REVIEW

File: setenv [hunk 1]
```diff
@@ -6,10 +6,11 @@
 
 CITADEL_ROOT="$(dirname $(readlink -f "${BASH_SOURCE[0]}"))"
 
+alias c="${CITADEL_ROOT}/bin/citadel"
```
Comment (AaronDewes) [inline setenv]: I'd rather not have this built-in, people may want to use c for something else. If power users want this, I think they can just add it themself.

File: cli/utils/functions.sh [hunk 2]
```diff
@@ -16,7 +16,7 @@ Commands:
     status                             Check the status of all services
     start                              Start the Citadel service
     stop                               Stop the Citadel service safely
-    restart                            Restart the Citadel service
+    restart <service>                  Restart Citadel or individual services
```
Comment (AaronDewes) [inline cli/utils/functions.sh]:
```suggestion
    restart [<service>]                  Restart Citadel or individual services
```

Not sure what the conventional way of specifying that a parameter is optional is, but I think we should do it somehow.

**Review by AaronDewes — APPROVED** [review]

File: setenv (hunk 1 above)
Comment (pwltr) [reply setenv]: Sure, I'll revert.

File: cli/utils/functions.sh (hunk 2 above)
Comment (pwltr) [reply cli/utils/functions.sh]: I looked at some man pages but can't really make out a clear convention. But it looks like optional args are always wrapped in just square brackets.

**Review by WilliamConnatser — APPROVED** [review]

**Pull request merged**